### DISCUSSION TITLE
Support driver for xclipse GPU

### DISCUSSION THREAD

**Discussion by hdk2d** [ptyfyre/mandarine-neo#113] [Citra-Enahced suggestions]
I'll just leave as a reminder if in the future they implement the driver for Exynos chipsets with AMD Xclipse GPU. 

Note, the Emulator runs smoothly with that GPU.![Screenshot_20240925_023256_Mandarine.jpg](https://github.com/user-attachments/assets/4f26b981-f029-4c68-aea9-5de0a3db1b37)

**Comment by LuisPerss** [comment]
and that the OpenGL and Vulkan APIs are also optimized for MediaTek Mali GPUs

**Reply by mxxme-dev** [reply]
xclipse has the richest driver on all mobile gpus

**Reply by hcrtrhn2-ship-it** [reply]
what?
